sql/src/test/java/io/crate/operation/collect/collectors/RemoteCollectBatchIteratorTest.java

```diff
@@ -57,20 +57,19 @@
 import java.util.UUID;
 
 import static io.crate.testing.TestingHelpers.createReference;
-import static org.hamcrest.Matchers.isA;
 import static org.mockito.Matchers.any;
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.*;
 
-public class RemoteCollectorTest {
+public class RemoteCollectBatchIteratorTest {
```

Comment:
No because mocking the TransportJobAction correctly to work for the tester is rather difficult.
I thought about creating a more decoupled generic BatchIteratorProxy but choose not to do that because it seemed overkill.